setup.py

```diff
@@ -52,89 +52,94 @@ def get_varstring(fname="mdtools/version.py", varname="__version__"):
     varname : str, optional
         Name of the variable whose value should be read.  The value must
         be a simple string.
-    
+
     Returns
     -------
     varstring : str
         The string assigned to `varname`.
-    
+
     Notes
     -----
     If `varname` is assigned multiple times, only the first assignment
     will be considered.
     """
-    with open(fname, 'r') as f:
+    with open(fname, "r") as f:
         for line in f:
             if line.startswith(varname):
                 if '"' in line:
                     delimiter = '"'
                 elif "'" in line:
                     delimiter = "'"
                 else:
-                    raise RuntimeError("The line starting with '{}' does"
-                                       " not contain a string"
-                                       .format(varname))
+                    raise RuntimeError(
+                        "The line starting with '{}' does"
+                        " not contain a string".format(varname)
+                    )
                 return line.split(delimiter)[1]
 
 
 def get_content(fname="README.rst"):
     """
-    Read a file.
-    
+    Read a text file.
+
     Parameters
     ----------
     fname : str, optional
         Path to the file to read.
-    
+
     Returns
     -------
     content : str
         The whole content of the file as one long string.
     """
-    with open(fname, 'r') as f:
+    with open(fname, "r") as f:
         return f.read()
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
     setuptools.setup(
         name=get_varstring("mdtools/_metadata.py", "__title__"),
         version=get_varstring("mdtools/version.py", "__version__"),
         author="Andreas Thum",
         author_email="andr.thum@gmail.com",
-        maintainer=get_varstring("mdtools/_metadata.py", "__maintainer__"),
-        maintainer_email=get_varstring("mdtools/_metadata.py", "__email__"),
+        maintainer=get_varstring(
+            "mdtools/_metadata.py", "__maintainer__"
+        ),
+        maintainer_email=get_varstring(
+            "mdtools/_metadata.py", "__email__"
+        ),
         url="https://github.com/andthum/mdtools",
         project_urls={
-            'Source': "https://github.com/andthum/mdtools",
-            'Documentation': "https://mdtools.readthedocs.io/en/latest/",
-            'Issue Tracker': "TODO",
-            'Feature Requests': "TODO",
+            "Source": "https://github.com/andthum/mdtools",
+            "Documentation": "https://mdtools.readthedocs.io/en/latest/",
+            "Issue Tracker": "TODO",
+            "Feature Requests": "TODO",
```

Comment:
I think one can link here `https://github.com/andthum/mdtools/issues` already instead of TODO.

Reply:
Good point.